lib/perl/Genome/Model/ReferenceAlignment.pm

```diff
@@ -260,17 +261,6 @@ sub _calculate_library_count {
     return scalar($self->libraries);
 }
 
-sub complete_build_directory {
```

Comment:
This was redundant to `last_complete_build_directory` on `ModelDeprecated` anyway!